pkg/provision/providers/docker/docker.go

```diff
@@ -85,11 +85,11 @@ func (p *provisioner) GetLoadBalancers(networkReq provision.NetworkRequest) (int
 	case "darwin", "windows":
 		return "", "127.0.0.1"
```

Comment:
I guess we'd want all of the platforms to return this actually?

Reply:
Unsure, why pollute local ports (also exposing them on 0.0.0.0) without a reason when we can just access those by IP. However a CLI option could by good (if it doesn't pollute opts)

Reply:
We should probably expose only on the localhost, and making things work same way across platforms makes it easier to test it. We don't have tests running on !Linux, so we won't be testing this code if we don't make it act same way.